pages/page.py

```diff
@@ -35,9 +34,7 @@ def __init__(self, testsetup):
     def is_the_current_page(self):
         if self._page_title:
             page_title = self.page_title
-            Assert.equal(page_title, self._page_title,
-                         "Expected page title: %s. Actual page title: %s" %
-                         (self._page_title, page_title))
+            assert page_title == self._page_title
```

Comment:
Just a nit, I'm still a bit bothered that our asserts no longer explain the failure and only report equality wasn't ascertained.